### PR TITLE
Loader fixes

### PR DESCRIPTION
**Pull request by kdutia**
Found a couple of bugs where the some metadata fields were wrongly split from the CSV, and posting documents from the loader DB to the API DB was failing.

### CODE REVIEW

**Review by joel-wright — APPROVED** [review]
LGTM

File: loader/app/poster/post.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def post_doc(db: Session, doc: Document) -> dict:
             "source_id": doc.source_id,
             "geography_id": doc.geography_id,  # this is from backend API lookup, so will exist remotely.
             "category_id": doc.category_id,  # this is from backend API lookup, so will exist remotely.
-            "publication_ts": doc.publication_ts,
+            "publication_ts": doc.publication_ts.isoformat(),
```
Comment (opyate) [inline loader/app/poster/post.py:103]: This is fixed in https://github.com/climatepolicyradar/navigator/pull/554 BTW :+1:

Comment (joel-wright) [reply loader/app/poster/post.py:103]: the new lines looked like they matched so I thought there would probably be no conflicts

Comment (joel-wright) [reply loader/app/poster/post.py:103]: :grin:

**Review by opyate — APPROVED** [review]

**Pull request merged**